flytekit/core/tracker.py

```diff
@@ -270,6 +270,9 @@ def _resolve_abs_module_name(self, path: str, package_root: typing.Optional[str]
         # Let us remove any extensions like .py
         basename = os.path.splitext(basename)[0]
 
+        if not Path(dirname).is_dir():
```

Comment:
since the executor will load the workflow from the zip file. 
![image](https://github.com/user-attachments/assets/5fdb03ef-e1cb-4ec2-a626-c475c9f74f7e)